common/test/common/ExperimentsTest.scala

```diff
@@ -11,9 +11,12 @@ class ExperimentsTest extends FlatSpec with Matchers {
 
   conf.switches.Switches.ServerSideExperiments.switchOn
 
-  "Experiments" should "not share participation group" in {
-    ActiveExperiments.allExperiments.size should be(ActiveExperiments.allExperiments.map(_.participationGroup).size)
-  }
+  // Nic Long said I was allowed to disable this to allow 2 50/50 AB tests to run for a certain period. Can be uncommented
+  // when AudioPageChanges is expired.
+
+//  "Experiments" should "not share participation group" in {
+//    ActiveExperiments.allExperiments.size should be(ActiveExperiments.allExperiments.map(_.participationGroup).size)
+//  }
```

Comment:
@nicl disabled this test and added this comment. Will make a card to go back and undo after test is set.
Test will run for a week. Will set the date on it once the PR is complete..

Reply:
and the Commercial test that was using this bucket has just been retired, will put the test back